config/manager/manager.yaml

```diff
@@ -33,6 +33,12 @@ spec:
         name: manager
         securityContext:
           allowPrivilegeEscalation: false
+          readOnlyRootFilesystem: true
+          runAsNonRoot: true
+          capabilities:
+            drop: [ "ALL" ]
+          seccompProfile:
+            type: RuntimeDefault
```

Comment:
The other one that seems to pop up when googling around is `runAsUser` https://kubesec.io/basics/containers-securitycontext-runasuser/ maybe thats trickier to align w/ the docker build etc? I'm not sure..

Reply:
Yeah, this is tricker, because the UID you run as has to exist, also it conflicts with some K8s distributions (OpenShift at least) which randomise the UIDs in the container